website/src/pages/index.astro

```diff
@@ -1,88 +1,134 @@
 ---
 import BaseLayout from '../layouts/base/BaseLayout.astro';
 import DataInfo from '../layouts/base/footer/DataInfo.astro';
+import { BrandName } from '../layouts/base/header/Brand';
 import { organismConfig } from '../types/Organism';
 import { dataOrigins } from '../types/dataOrigins';
 import { Page } from '../types/pages';
 import { ServerSide } from '../views/serverSideRouting';
+
+/* eslint-disable @typescript-eslint/naming-convention */
+const iconMapping: { [key: string]: string } = {
+    'Sequencing efforts': 'mdi--test-tube',
+    'Single variant': 'mdi--magnify',
+    'Compare side-by-side': 'mdi--compare-horizontal',
+    'Browse data': 'mdi--database-outline',
+};
+/* eslint-enable */
 ---
 
 <style>
+    a {
+        @apply text-slate-500 hover:underline;
+    }
+
+    .items a {
+        @apply text-black hover:underline hover:decoration-4;
+    }
```

Comment:
What is this part doing? If I remove it, then I don't see a difference in the browser (maybe I'm missing something though)

Reply:
It keeps the color of the names of the links in the pathogen cards black and makes sure that their colorful underlines are thicker